File: src/Models/Region.java
package Models;

public class Region implements HasID{
    public int region_id;
    public String region_name;
    public String country;

    public Region(int region_id, String region_name, String country) {
        this.region_id = region_id;
        this.region_name = region_name;
        this.country = country;
    }

    public int getID() {
        return region_id;
    }

    public void setRegion_id(int region_id) {
        this.region_id = region_id;
    }

    public String getRegion_name() {
        return region_name;
    }

    public void setRegion_name(String region_name) {
        this.region_name = region_name;
    }

    public String getCountry() {
        return country;
    }

    public void setCountry(String country) {
        this.country = country;
    }

    @Override
    public String toString() {
        return "Region{" +
                "region_id=" + region_id +
                ", region_name='" + region_name + '\'' +
                ", country='" + country + '\'' +
                '}';
    }
}

File: Python code/Entities/Region.py

class Region:
    def __init__(self, region_id,name, country):
        self.name = name
        self.region_id = region_id
        self.country=country

    def get_region_id(self):
        return self.region_id

    def get_name(self):
        return self.name

    def get_country(self):
        return self.country

    def set_region_id(self, region_id):
        self.region_id = region_id

    def set_name(self, name):
        self.name = name

    def set_country(self, country):
        self.country = country

    def __str__(self):
        return f"Region: {self.name} ({self.country})"